### ISSUE TITLE
PCF File maybe incorrect

### ISSUE THREAD

**Issue by wd5gnr** [tomverbeure/upduino#2]
I was playing with the board and assuming nothing has changed, I show LED_G is pin 39 and LED_B is 40. Of  course, you turn them all on so you wouldn't notice, but if you jam two of them as 1'b1 and pass led to the 3rd one you'll see that G and B are swapped.

**Comment by wd5gnr** [comment]
Just an update. Looking at the schematic, it should be BLUE=39 (RGB2), Green=40 (RGB1), and Red=41 (RGB0). However, this is not the case on my board. It is R=41 (check), G=39 and B=40. 

This makes a difference if you are using the RGB peripheral, as well, since the code expects RGB0 to connect to the right pin. So I don't know if the schematic is wrong or my board has a different brand LED on it or what.

Bottom line is your PCF matches the schematic but not the board I have in my hand.

**Comment by tomverbeure** [comment]
Feel free to create a pull request with the fix.

The Upduino is one of 7 FPGA board that I currently have at home, and it's unlikely to get tender loving care anytime soon!

**Comment by tomverbeure** [comment]
My board matches what you're seeing on your board.

I have corrected the .pcf file.

Closing.

Tom
